pallets/identity/src/lib.rs

```diff
@@ -1653,10 +1649,8 @@ impl<T: Trait> Module<T> {
     pub fn can_key_be_linked_to_did(key: &T::AccountId) -> bool {
         if <KeyToIdentityIds<T>>::get(key).is_some() {
             false
-        } else if T::MultiSig::is_signer(key) {
-            false
         } else {
-            true
+            T::MultiSig::is_signer(key)
```

Comment:
You are missing a negation here. The return should be `!T::MultiSig::is_signer(key)`